plugins/modules/packaging/os/yum_versionlock.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2018, Florian Paul Hoberg <florian.hoberg@credativ.de>
+# Written by Florian Paul Hoberg <florian.hoberg@credativ.de>
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: yum_versionlock
+version_added: 2.7
+short_description: Locks/Unlocks an installed package(s) from being updates by (yum) package manager.
+description:
+     - This module adds installed packages to yum versionlock to prevent it from being updated.
+options:
+  package:
+    description:
+      - Package name(s) like C(httpd). Multiple packages are supported by adding them by whitespaces.
```

Comment:
```suggestion
      - Package name or a list of packages.
      type: list
```
imo, more logical to have a list here.
So we could pass values separated by commas or using the yml list syntax (see the changed example).